raiden/blockchain/events.py

```diff
@@ -135,6 +138,27 @@ def get_all_netting_channel_events(
     )
 
 
+def get_all_secret_registry_events(
+        chain: BlockChainService,
+        secret_registry_address: Address,
+        events: List[str] = ALL_EVENTS,
+        from_block: BlockSpecification = 0,
+        to_block: BlockSpecification = 'latest',
+) -> List[Dict]:
+    """ Helper to get all events of a SecretRegistry. """
+
+    return get_contract_events(
+        chain,
+        raiden_contracts.contract_manager.CONTRACT_MANAGER.get_contract_abi(
```

Comment:
why are we using a different contract manager to get the abi here?

Reply:
It's just temporary solution until the new contracts are integrated (to keep old & new contracts separated). When we switch over there'll be a single instance.